test/compositing.test.js

```diff
@@ -38,25 +38,24 @@ test('should stitch input images correctly', (assert) => {
     output_image.composite(item[0], {dx: item[1], dy: item[2]}, function(err) {
       if (err) throw err;
       assert.ok(output_image);
-    });
-  });
-  var views = [output_image.view(0, 0, 256, 256),
-               output_image.view(256, 0, 256, 256),
-               output_image.view(256, 256, 256, 256),
-               output_image.view(0, 256, 256, 256)];
+      var views = [output_image.view(0, 0, 256, 256),
+                   output_image.view(256, 0, 256, 256),
+                   output_image.view(256, 256, 256, 256),
+                   output_image.view(0, 256, 256, 256)];
 
-  views.forEach(function(view, index) {
-    var equals = true;
-    for (var x = 0; x < view.width(); ++x) {
-      for (var y = 0; y < view.height(); ++y) {
-        if (view.getPixel(x, y) != input[index][0].getPixel(x, y)) {
-          equals = false;
-          break;
+      views.forEach(function(view, index) {
+        var equals = true;
+        for (var x = 0; x < view.width(); ++x) {
+          for (var y = 0; y < view.height(); ++y) {
+            if (view.getPixel(x, y) != input[index][0].getPixel(x, y)) {
+              equals = false;
+              break;
+            }
+          }
```

Comment:
Could you add a comment above this test explaining what it is trying to test? e.g. if it is a stress test trying to do something dodgy but that should not crash, would be good to know that. Vs if it is trying to do something legit (e.g. that will be determinative). As it is currently written the `output_image.composite` looks non-determinitive because it is async and called multiple times in a loop, such that it's unclear which thread will finish writing to `output_image` first.